src/core/lib/security/credentials/jwt/json_token.cc

```diff
@@ -175,15 +176,19 @@ static char* encoded_jwt_claim(const grpc_auth_json_key* json_key,
 
   Json::Object object = {
       {"iss", json_key->client_email},
-      {"aud", audience},
       {"iat", now.tv_sec},
       {"exp", expiration.tv_sec},
   };
+
   if (scope != nullptr) {
```

Comment:
How about having something like this:
```
if (scope != nullptr && enable_scope_for_jwt) {
  object["scope"] = scope
} else {
  object["aud"] = audience;
}
```

Reply:
By doing that, it does not allow to have both `scope` and `audience` fields. Please see my above comment.